src/test/unit/profile-store.test.js

```diff
@@ -167,54 +166,44 @@ describe('profile upload', () => {
 });
 
 describe('profile deletion', () => {
-  beforeEach(() => {
-    window.fetch = jest.fn();
-  });
-
-  afterEach(() => {
-    delete window.fetch;
-  });
-
   function mockFetchForDeleteProfile({
     endpointUrl,
     jwtToken,
   }: {
     endpointUrl: string,
     jwtToken: string,
   }) {
-    window.fetch.mockImplementation(async (urlString, options) => {
-      if (urlString !== endpointUrl) {
-        return new Response(null, { status: 404, statusText: 'Not found' });
-      }
-
-      const { method, headers } = options;
-
-      if (method !== 'DELETE') {
-        return new Response(null, {
-          status: 405,
-          statusText: 'Method not allowed',
-        });
-      }
+    window.fetch
```

Comment:
Similarly to the test for `ListOfPublishedProfiles.test.js` this isn't wildly different than what it was before.